src/classes/frFundraisingEventTest.cls

```diff
@@ -0,0 +1,209 @@
+/*
+ *
+ *  Copyright (c) 2019, Funraise Inc
+ *  All rights reserved.
+ *
+ *  Redistribution and use in source and binary forms, with or without
+ *  modification, are permitted provided that the following conditions are met:
+ *  1. Redistributions of source code must retain the above copyright
+ *     notice, this list of conditions and the following disclaimer.
+ *  2. Redistributions in binary form must reproduce the above copyright
+ *     notice, this list of conditions and the following disclaimer in the
+ *     documentation and/or other materials provided with the distribution.
+ *  3. All advertising materials mentioning features or use of this software
+ *     must display the following acknowledgement:
+ *     This product includes software developed by the <organization>.
+ *  4. Neither the name of the <organization> nor the
+ *     names of its contributors may be used to endorse or promote products
+ *     derived from this software without specific prior written permission.
+ *
+ *  THIS SOFTWARE IS PROVIDED BY FUNRAISE INC ''AS IS'' AND ANY
+ *  EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+ *  WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ *  DISCLAIMED. IN NO EVENT SHALL FUNRAISE INC BE LIABLE FOR ANY
+ *  DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ *  (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ *  LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ *  ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ *  (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ *  SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ *
+ *
+ *
+ * PURPOSE:
+ *
+ *
+ *
+ * CREATED: 2019 Funraise Inc - https://funraise.io
+ * AUTHOR: Alex Molina
+ */
+@isTest
+public class frFundraisingEventTest {  
+    static testMethod void createEvent_throughWS() {
+        String funraiseId = '25';
+        Map<String, Object> request = new Map<String, Object>();
+        List<Object> startDate = new List<Object> {2019, 05, 11, 9, 0};
+        List<Object> endDate = new List<Object> {2019, 05, 11, 4, 30};
```

Comment:
The end date does not seem to be syncing over for me. Could it be related to the missing seconds and milliseconds?
![image](https://user-images.githubusercontent.com/50249863/57654524-387a0180-7589-11e9-8501-45cdaf799067.png)

```http
POST http://{{host}}/api/v2/fundraising-event
Accept: */*
Cache-Control: no-cache
X-AUTH-TOKEN: {{auth_token}}
Content-Type: application/json

{
  "name": "does the end date work",
  "description": "string",
  "locationName": "string",
  "address": "string",
  "city": "string",
  "country": "string",
  "state": "string",
  "postalCode": "string",
  "eventStartDate": "2018-05-07T20:38:28.182Z",
  "eventEndDate": "2018-05-08T20:38:28.182Z"
}

> {%
  client.global.set('fundraising_event_id', response.body.id);
%}

###
```

![image](https://user-images.githubusercontent.com/50249863/57654659-9a3a6b80-7589-11e9-98fa-733bae62187a.png)

Reply:
Hm yeah, where did you get that sample request? When I checked, the format that came over (because it's a LocalDateTime in the DTO) was an array rather than the format "2018-05-08T20:38:28.182Z"

Here's a sample request from my local dev
`{deleted=false, description=123456, eventEndDate=(2019, 1, 25, 12, 30), eventStartDate=(2019, 1, 25, 12, 0), id=1, name=Test Event}`

Reply:
Yep the example request I provided was one generated by swagger which appears to add the milliseconds. If I used the same format as the front-end (by using chrome's dev console and inspecting the request body), then we get the desired result. This is probably fine if you don't want to make any changes.